charts/alpha/keycloak/README.md

```diff
@@ -0,0 +1,62 @@
+configures [Keycloak](https://github.com/bitnami/charts/tree/main/bitnami/keycloak).
+
+## Image
+
+You will need to mirror the [Keycloak image](https://hub.docker.com/r/bitnami/keycloak/tags) to your private containers registry and update the chart's image reference accordingly.
+See the section below for the relevant configuration overrides to do so.
+
+## Configuration
+
+> Refer to the Keycloak [README](https://github.com/bitnami/charts/tree/main/bitnami/keycloak#parameters) for all
+> available upstream configuration options
+
+### General Configuration
+
+| Parameter                               | Description                                                                       | Default                                     | Notes                                                                                                    |
+|-----------------------------------------|-----------------------------------------------------------------------------------|---------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| `tls.keystorePassword` (**Required**)   | Password to access the keystore when it's password-protected                      | `""`                                        | See the FedStart documentation for details                                                               |
+| `tls.truststorePassword` (**Required**) | Password to access the truststore when it's password-protected                    | `""`                                        | See the FedStart documentation for details                                                               |
+
+### Database Configuration
+
+| Parameter                         | Description                                                                        | Default  | Notes                                                                                                                                                                                                                                                                                 |
+|-----------------------------------|------------------------------------------------------------------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `postgresql.enabled`              | Switch to true to enable the PostgreSQL helm chart instead of using an external DB | `false`  | To ensure your Keycloak deployment is highly available, it's recommended that you use an external DB instance. Set up this connection using the overrides under the [`externalDatabase`](https://github.com/bitnami/charts/tree/main/bitnami/keycloak#database-parameters) parameter. |
```

Comment:
do we need to document this? Do we actually want to support using postgresql at all?